### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Zenjjim**
# Update docker and testing
## Description
Link to Trello task: ...

Comments/issues:

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [ ] The PR includes the link to the relevant Trello task
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [ ] `python manage.py runserver` was run locally witout errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -1,20 +1,29 @@
 ## Start the webserver with docker on http://localhost:8000
```
Comment (Andorr) [inline makefile]: Skal være "Makefile" nerd

File: app/settings.py [hunk 2]
```diff
@@ -94,11 +94,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': os.environ.get('DATABASE_NAME'),
-        'USER': os.environ.get('DATABASE_USER'),
-        'PASSWORD': os.environ.get('DATABASE_PASSWORD'),
-        'HOST': os.environ.get('DATABASE_HOST'),
-        'PORT': os.environ.get('DATABASE_PORT'),
+        'NAME': os.environ.get('DATABASE_DEV_NAME'),
```
Comment (Andorr) [inline app/settings.py]: Wut is dis?

Comment (Zenjjim) [reply app/settings.py]: det er kobling opp mot docker database

Comment (Andorr) [reply app/settings.py]: Nerd 🐰

Comment (Zenjjim) [reply app/settings.py]: men blir sykt chill med tanke på migrations

Comment (Andorr) [reply app/settings.py]: Skjønner det da, men migrations er for folk som ikke kan skrive SQL. Jeg pleier alltid å connecte til databasen og skrive inn queriene selv. Får bedre performance da. Har ikke så mye å si siden det er så få brukere da.

💋

**Pull request merged**